drivers/bridge/bridge_store.go

```diff
@@ -184,7 +184,7 @@ func (ncfg *networkConfiguration) Exists() bool {
 }
 
 func (ncfg *networkConfiguration) Skip() bool {
-	return ncfg.DefaultBridge
+	return false
```

Comment:
@chenchun  The most changes for this is on daemon side, I checked if there is old running containers, if is, then don't remove the default bridge network and throw a warning, and if no, then remove the default bridge network.
````
        controller, restored, err := libnetwork.New(oldRunningContainers, netOptions...)
        if err != nil {
                return nil, fmt.Errorf("error obtaining controller instance: %v", err)
        }
        if len(restored) > 0 {
                logrus.Warnf("There are old running containers, the network config will not take affect")
                return controller, nil
        }

````